types/stylelint/index.d.ts

```diff
@@ -66,7 +67,7 @@ declare module 'stylelint' {
 	}[keyof T];
 	export type DisablePropertyName = PropertyNamesOfType<StylelintConfig, DisableSettings>;
 
-	export type CosmiconfigResult = { config: StylelintConfig; filepath: string };
+	export type StylelintCosmiconfigResult = (CosmiconfigResult & { config: StylelintConfig }) | null;
```

Comment:
The original `CosmiconfigResult` type also [includes `| null`](https://github.com/davidtheclark/cosmiconfig/blob/v7.0.1/src/types.ts#L10).